fs/config/rc.go

```diff
@@ -228,16 +228,24 @@ func init() {
 Parameters:
```

Comment:
Should this be a separate rc call `config/setpassword` maybe? As you might want to set the password but not change the path?

Reply:
I prefer the separation of concerns - one API call for each thing.

Can you split it into its own API please?

Reply:
Done.